tests/utilities/test_ast_from_value.py

```diff
@@ -86,6 +86,10 @@ def converts_float_values_to_float_asts():
 
         assert ast_from_value(1e40, GraphQLFloat) == FloatValueNode(value="1e+40")
 
+        assert ast_from_value(123456789.0123456, GraphQLFloat) == FloatValueNode(
+            value="123456789.0123456"
+        )
+
```

Comment:
Can you add another test for `1.1` so that we avoid the ugly representation that we would get with `.17f` format?

Reply:
Will add that test, as well as a test that encodes the `1.0` -> `1` serialization expectation as well. With the change to `str` serialization, I cal also change this test to assert 17 digits of precision.